### PR TITLE
docs(fr): new version 3 translation

### PR DESCRIPTION
**Pull request by Geminii**
_Nuxt3 page :_
- [x] nuxt3 coming soon

_Deployment pages :_
- [x] 21yunbox deployment
- [x] amazon web services deployment
- [x] azure portal deployment
- [x] bip deployment
- [x] cleavr deployment
- [x] app platform DigitalOcean deployment

_Code examples pages :_ 
- [x] routing code examples
- [x] data fetching code examples
- [x] assets management code examples
- [x] transitions code examples
- [x] seo code examples
- [x] loading code examples
- [x] miscellaneous code examples
- [x] middlewares code examples
- [x] plugins examples
- [x] modules examples

_Resources page :_ 
- [x] Video courses

Reminder : Let's remember to update all the codesandbox of examples with Nuxt3 😇

### CODE REVIEW

File: content/fr/v3.md [hunk 1]
```diff
@@ -13,134 +13,182 @@ head.titleTemplate: null
 ::nuxt3-hero
 primary: false
 ---title
-Nuxt 3 is coming[.]{.text-primary}
+Nuxt 3 arrive[.]{.text-primary}
 ---description
-Rebuilt from the ground up with TypeScript and a stronger architecture.<br>
-Nuxt 3 is bringing web application to a new level, hybrid static & server rendering, api routes, serverless.
+Reconstruit à partir de zéro avec TypeScript et une architecture plus solide.<br>
+Nuxt 3 amène les applications web à un nouveau niveau, rendu hybride statique et serveur, routes API, sans serveur.
```
Comment (Atinux) [inline content/fr/v3.md:19]: Je pense pas que serverless se traduit en "sans serveur"

Comment (Geminii) [reply content/fr/v3.md:19]: Yes, il ne se traduit pas. Je l'ai déjà corrigé dans ma nouvelle version qui arrive :)

File: content/fr/1.examples/6.loading/2.customize-loading-indicator.md [hunk 2]
```diff
@@ -1,23 +1,23 @@
 ---
-title: Customize Nuxt Loading Indicator
-description: Customize the Nuxt Loading Indicator for when ssr is set to false
+title: Personnaliser l'indacteur de chargement de Nuxt
```
Comment (clemcode) [inline content/fr/1.examples/6.loading/2.customize-loading-indicator.md:2]:
```suggestion
title: Personnaliser l'indicateur de chargement de Nuxt
```

File: content/fr/1.examples/7.miscellaneous/2.lazy-loading-components.md [hunk 3]
```diff
@@ -1,27 +1,27 @@
 ---
-title: Lazy Loading Components
-description: Use fetch in your components to fetch data from an API as well as auto importing and lazy loading components
+title: Chargement de composants en arrière plan
+description: Utilisez fetch dans vos composants pour récupérer des données à partir d'une API ainsi que d'importer automatiquement vos composants et de chargement différé.
```
Comment (clemcode) [inline content/fr/1.examples/7.miscellaneous/2.lazy-loading-components.md:3]:
```suggestion
description: Utilisez fetch dans vos composants pour récupérer des données à partir d'une API ainsi qu'importer automatiquement vos composants en chargement différé.
```

File: content/fr/1.examples/7.miscellaneous/2.lazy-loading-components.md [hunk 4]
```diff
@@ -1,27 +1,27 @@
...
 category: miscellaneous
 ---
-# Lazy Loading Components
+# Chargement de composants en arrière plan
 
-Use fetch in your components to fetch data from an API as well as auto importing and lazy loading components
+Utilisez fetch dans vos composants pour récupérer des données à partir d'une API ainsi que d'importer automatiquement vos composants et de chargement différé.
```
Comment (clemcode) [inline content/fr/1.examples/7.miscellaneous/2.lazy-loading-components.md:8]:
```suggestion
Utilisez fetch dans vos composants pour récupérer des données à partir d'une API ainsi qu'importer automatiquement vos composants en chargement différé.
```

File: content/fr/1.examples/8.middlewares/1.router.md [hunk 5]
```diff
@@ -1,26 +1,26 @@
 ---
-title: Router Middleware
-description: Using router middleware to set a class to the body so we can then style differently depending on the route
+title: Routeur de middleware
```
Comment (clemcode) [inline content/fr/1.examples/8.middlewares/1.router.md:2]:
```suggestion
title: Middleware de routage
```

File: content/fr/1.examples/8.middlewares/1.router.md [hunk 6]
```diff
@@ -1,26 +1,26 @@
...
+description: Utiliser un middleware de routeur pour définir une classe pour le corps afin que nous puissions ensuite le styliser différemment en fonction de la route.
 category: middleware
 ---
-# Router Middleware
+# Routeur de middleware
```
Comment (clemcode) [inline content/fr/1.examples/8.middlewares/1.router.md:6]:
```suggestion
# Middleware de routage
```

File: content/fr/1.examples/8.middlewares/1.router.md [hunk 7]
```diff
@@ -1,26 +1,26 @@
 ---
-title: Router Middleware
-description: Using router middleware to set a class to the body so we can then style differently depending on the route
+title: Routeur de middleware
+description: Utiliser un middleware de routeur pour définir une classe pour le corps afin que nous puissions ensuite le styliser différemment en fonction de la route.
```
Comment (clemcode) [inline content/fr/1.examples/8.middlewares/1.router.md:3]:
```suggestion
description: Utilisez un middleware de routage pour définir une classe pour le corps afin que nous puissions ensuite le styliser différemment en fonction de la route.
```

File: content/fr/1.examples/8.middlewares/1.router.md [hunk 8]
```diff
@@ -1,26 +1,26 @@
...
 category: middleware
 ---
-# Router Middleware
+# Routeur de middleware
 
-Using router middleware to set a class to the body so we can then style differently depending on the route
+Utiliser un middleware de routeur pour définir une classe pour le corps afin que nous puissions ensuite le styliser différemment en fonction de la route.
```
Comment (clemcode) [inline content/fr/1.examples/8.middlewares/1.router.md:8]:
```suggestion
Utiliser un middleware de routage pour définir une classe pour le corps afin que nous puissions ensuite le styliser différemment en fonction de la route.
```

File: content/fr/1.examples/8.middlewares/1.router.md [hunk 9]
```diff
@@ -1,26 +1,26 @@
...
 ---
 
-In this example:
+Dans cet exemple :
 
-`store/class.js` sets a class to the body.
+`store/class.js` définit une classe sur le corps.
 
-`middleware/class.js` uses router middleware to set a class before we enter the route.
+`middleware/class.js` utilise un middleware de routeur pour définir une classe avant d'entrer dans la route.
```
Comment (clemcode) [inline content/fr/1.examples/8.middlewares/1.router.md:16]:
```suggestion
`middleware/class.js` utilise un middleware de routage pour définir une classe avant d'entrer dans la route.
```

File: content/fr/1.examples/8.middlewares/2.named.md [hunk 10]
```diff
@@ -1,26 +1,26 @@
 ---
-title: Named Middleware
-description: Using named middleware to authenticate a user using the store and allow them to visit a page once authenticated
+title: Middleware nommés
```
Comment (clemcode) [inline content/fr/1.examples/8.middlewares/2.named.md:2]:
```suggestion
title: Middleware nommé
```

File: content/fr/1.examples/8.middlewares/2.named.md [hunk 11]
```diff
@@ -1,26 +1,26 @@
...
+description: Utiliser un middleware nommé pour authentifier un utilisateur utilisant le store et lui permettre de visiter une page une fois authentifié.
 category: middleware
 ---
-# Named Middleware
+# Middleware nommés
```
Comment (clemcode) [inline content/fr/1.examples/8.middlewares/2.named.md:6]:
```suggestion
# Middleware nommé
```

File: content/fr/3.resources/video-courses.md [hunk 12]
```diff
@@ -13,49 +13,49 @@ description: "Apprenez-en plus sur le framework Nuxt avec ce choix de vidéos en
...
+    title: Démarrer avec Nuxt
+    description: Apprenez l'essentiel pour créer et déployer un site Nuxt, y compris les routes dynamiques, la récupération de données, le référencement, le chargement en arrière plan, les styles globaux et les transitions, ainsi que comment générer et déployer votre application Nuxt.
     image: /img/video-courses/jamstack_explorer.png
-    tier: Free
+    tier: Gratuit
   three:
-    title: Learn Nuxt with Debbie
-    description: A playlist of YouTube videos covering all things Nuxt including short videos and live streams.
+    title: Apprenez Nuxt avec Debbie
+    description: Une liste de lecture de vidéos YouTube couvrant tout ce qui concerne Nuxt, y compris de courtes vidéos et des flux en direct.
```
Comment (clemcode) [inline content/fr/3.resources/video-courses.md:27]:
```suggestion
    description: Une liste de lecture de vidéos YouTube couvrant tout ce qui concerne Nuxt, y compris de courtes vidéos et des livestreams.
```

File: content/fr/v3.md [hunk 13]
```diff
@@ -13,134 +13,182 @@ head.titleTemplate: null
...
-After more than 16 months of work, Nuxt 3 is ready to be released, in three stages.
+Après plus de 16 mois de travail, Nuxt 3 est prêt à sortir, en trois étapes.
 ---
 
   ::section-content-item
   image: /img/home/learn/guides/gem-usage.svg
-  title: July 2021
-  description: Private beta for Vue core team, Nuxt modules authors and partners.
+  title: Juillet 2021
+  description: Bêta privée pour l'équipe principale de Vue et les auteurs de modules Nuxt.
```
Comment (clemcode) [inline content/fr/v3.md:35]:
```suggestion
  description: Bêta privée pour l'équipe core de Vue et les auteurs de modules Nuxt.
```

File: content/fr/v3.md [hunk 14]
```diff
@@ -13,134 +13,182 @@ head.titleTemplate: null
...
+Avec de nouvelles fonctionnalités
 ---description
-Rebuilt from the ground up with TypeScript and a stronger architecture, Nuxt 3 offers a smaller core and a set of new features for performance and developer experience.
+Reconstruit à partir de zéro avec TypeScript et une architecture plus solide, Nuxt 3 offre un noyau plus petit et un ensemble de nouvelles fonctionnalités pour les performances et l'expérience des développeurs.
 ---
   ::section-content-item
-  title: Lighter
-  description: The default Nuxt 3 app is 13kb, it's 20% reduction with Nuxt 2.
+  title: Léger
+  description: "L'application Nuxt 3 par défaut est de 13 Ko, soit une réduction de 20% par rapport à Nuxt 2."
```
Comment (clemcode) [inline content/fr/v3.md:61]:
```suggestion
  description: "L'application Nuxt 3 pèse par défaut 13 Ko, soit une réduction de 20% par rapport à Nuxt 2."
```

File: content/fr/v3.md [hunk 15]
```diff
@@ -13,134 +13,182 @@ head.titleTemplate: null
...
   image: IconFeather
   imageClass: w-10 h-10
   ---
   :icon-nuxt
   ::
   ::section-content-item
-  title: Faster
-  description: 'Faster server bootup and response time with code-splitting, powered by h3.'
+  title: Rapide
+  description: 'Démarrage du serveur et temps de réponse plus rapides grâce au fractionnement de code, optimisé par h3.'
```
Comment (clemcode) [inline content/fr/v3.md:69]:
```suggestion
  description: 'Démarrage du serveur et temps de réponse plus rapides grâce au code-splitting, optimisé par h3.'
```

File: content/fr/v3.md [hunk 16]
```diff
@@ -13,134 +13,182 @@ head.titleTemplate: null
...
   image: IconRabbit
   imageClass: w-10 h-10
   ---
   ::
   ::section-content-item
-  title: Hybrid
-  description: 'Incremental Static Generation is now possible, specify your static pages and let Nuxt handle the rest.'
+  title: Hybride
+  description: "La génération statique incrémentielle est désormais possible, spécifiez vos pages statiques et laissez Nuxt s'occuper du reste."
```
Comment (clemcode) [inline content/fr/v3.md:76]:
```suggestion
  description: "La génération statique incrémentale est désormais possible, spécifiez vos pages statiques et laissez Nuxt s'occuper du reste."
```

**Review by clemcode — CHANGES_REQUESTED** [review]

File: content/fr/v3.md [hunk 17]
```diff
@@ -13,134 +13,182 @@ head.titleTemplate: null
...
+
+::section{.dark:bg-secondary-darkest .bg-gray-50 .py-20 .text-lg}
+  ::nuxt-container
+  :headline[Moteur Nitro]
+
+  Nous avons travaillé pendant 9 mois sur le nouveau moteur de serveur pour Nuxt : **Nitro**. Il débloque de nouvelles **capacités robustes** sur le serveur Nuxt et au-delà.
+  
+  En développement, il utilise des noeuds de calcul et des noeuds pour fournir un remplacement de module à chaud pour votre code serveur et l'isolation du contexte. Il **génére également l'API de votre serveur** en lisant les fichiers dans `server/api/` et des **fonctions serveurs** à partir de `server/functions/`.
+  
+  En production, il construit votre application et votre serveur dans un répertoire universel `.output`. Cette **sortie est légère** : minimisée et supprimée de tous les modules de node (attendez les polyfills). Vous pouvez déployer cette sortie sur n'importe quel système prenant en charge JavaScript, à partir de Node, Serverless, Workers, Edge-side Rendering ou purement statique.
```
Comment (clemcode) [inline content/fr/v3.md:157]:
```suggestion
  En production, il construit votre application et votre serveur dans un répertoire universel `.output`. Cette **sortie est légère** : minimisée et supprimée de tous les modules de node (excepté les polyfills). Vous pouvez déployer cette sortie sur n'importe quel système prenant en charge JavaScript, à partir de Node, Serverless, Workers, Edge-side Rendering ou purement statique.
```

**Review by clemcode — APPROVED** [review]
👍

**Pull request merged**